app.py

```diff
@@ -89,13 +85,13 @@ def get_members_by_state(state_):
                                                  role = result['role'],
                                                  last_name =result['last_name']),
                                                  house_results)
-                          
```

Comment:
Now we need to refactor the MemberIndex initialize method in the same way that we did the MemberShow initialize.